devito/pointdata.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, *args, **kwargs):
             self.coordinates = DenseData(name='%s_coords' % self.name,
                                          dimensions=[self.indices[1], d],
```

Comment:
Best if @mlange05 answer since he wrote this code but I suppose this is the index for the dimension that counts the number of sparse points contained in a `PointData` instance. Sometimes this dimension is explicitly provided, sometimes implicitly assumed because `npoints` was passed.